packages/material/scss/menu/_theme.scss

```diff
@@ -28,8 +31,6 @@ $nav-selected-text: $base-text !default;
     .k-menu-scroll-wrapper {
 
         .k-menu-scroll-button {
-            color: inherit;
```

Comment:
This rule is necessary to prevent changing color by hover in Material

Reply:
This exact property is now generated by theme-default, thanks to `$menu-scrolling-icon: inherit`. We can set `$menu-scrolling-icon-hover: inherit` to remove the hover effect, without the need for overrides.